modules/search/search.py

```diff
@@ -1,8 +1,21 @@
 import json
 import urllib2
-
+import re
 ytkey = "AIzaSyDRNhlYHDjCTo_5AnAuYHbSb53YbnmQIRs"
 
+def YTDurationToSeconds(duration):
+  match = re.match('PT(\d+H)?(\d+M)?(\d+S)?', duration).groups()
+  hours = _js_parseInt(match[0]) if match[0] else 0

```

Comment:
I think this should be removed: we only need minutes/seconds I think. It looks better on the UI that way too because then the text won't overflow from the time boxes